x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/session/EsqlCCSUtils.java

```diff
@@ -269,8 +269,8 @@ static boolean concreteIndexRequested(String indexExpression) {
     // visible for testing
     static void updateExecutionInfoAtEndOfPlanning(EsqlExecutionInfo execInfo) {
         // TODO: this logic assumes a single phase execution model, so it may need to altered once INLINESTATS is made CCS compatible
+        execInfo.markEndPlanning();
         if (execInfo.isCrossClusterSearch()) {
-            execInfo.markEndPlanning();
```

Comment:
Was this a bug? Was it working when not CCS?

Reply:
Not really. 
This method was introduced with CCS implementation, and `planningTookTime` was only used in that contex.
Now we need it also for other purposes, so I'll set it also for non-CCS queries.